tensorflow/core/kernels/crop_and_resize_op.cc

```diff
@@ -34,13 +34,18 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/work_sharder.h"
 
-#if GOOGLE_CUDA
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 #include "tensorflow/core/common_runtime/gpu/gpu_event_mgr.h"
-#include "tensorflow/core/platform/cuda.h"
 #include "tensorflow/core/platform/stream_executor.h"
+#endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 
+#if GOOGLE_CUDA
```

Comment:
Maybe we should put these lines in a shared place. If it's not too much trouble, and in a separate PR.

Reply:
I looked around a little bit for this. It looks like `ScopedActivateExecutorContext` in either [cuda::ScopedActivateExecutorContext](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/stream_executor/cuda/cuda_activation.h#L34), or [rocm::ScopedActivateExecutorContext](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/stream_executor/rocm/rocm_activation.h#L34), are just alias to [gpu::ScopedActivateExecutorContext](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/stream_executor/gpu/gpu_activation.h#L38). Is it easier to just move the corresponding `using stream_executor::*::ScopedActivateExecutorContext` to either `rocm_activation.h` or `cuda_activation.h`?